Update customize-image.md (#1959)
astronomer/docs: software/customize-image.md

> **Note:** The guidelines below assume that you've initialized a project on Astronomer via `$ astro dev init`. If you haven't done so already, refer to our ["CLI Quickstart" doc](https://docs.astronomer.io/astro/cli/install-cli).

## Add Python and OS-level dependencies

> To build Python and OS-level packages into your Airflow Deployment, add them to the deployment's `requirements.txt` and `packages.txt` files, respectively. Both files were automatically generated when you initialized an Astro project locally via `$ astro dev init`.

Reviewer: ```suggestion
To build Python and OS-level packages into your Airflow Deployment, add the packages to the Deployment `requirements.txt` and `packages.txt` files. These files were automatically generated when you ran `$ astro dev init` to initialize your local Astro project.
```

Author: I think we should say something like "respectively" or otherwise indicate that Python packages -> requirements.txt and OS packages -> packages.txt. They cannot be mixed, and I think we should make that clear

Reviewer: @RNHTTR I've made an adjustment to the proposed text. I think it's clearer now.

### Add Python dependencies

> Add all Python packages to your `requirements.txt` file and they will be automatically built into your Airflow deployment.

Reviewer: ```suggestion
To automatically add all Python packages to an Airflow Deployment, add the packages to the Deployment `requirements.txt` file.
```

If you do _not_ pin a package to a version, the latest version of the package that's publicly available will be installed by default.

### Add OS-level dependencies

> Add any OS-level packages you'd like to include to your `packages.txt` file.

Reviewer: ```suggestion
To include OS-level packages with a Deployment, add them to the Deployment `packages.txt` file.
```
